### DISCUSSION TITLE
Release-5.2.0 Zookeeper provision issue( Provision/dev/DataPipeline/Zookeeper)

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#547] [Bugs] [answered]
Hi team!
@santhosh-tg 
zookeeper package (3.6.3)is not available in the official website, updated the version.
https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/pull/1619

Regards,
Gohila

**Comment by santhosh-tg** [comment]
@anandp504 @manjudr  Please check on this

**Comment by gohilamariappan** [comment]
@santhosh-tg Kindly merge this PR
https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/pull/1621

**Reply by santhosh-tg** [reply, marked as answer]
@gandham-santhosh  As discussed, the PR is updated. Please check
